src/ui/qgspgnewconnectionbase.ui

```diff
@@ -265,7 +265,7 @@
 &lt;body&gt;
 &lt;p&gt;When the layer is setup various metadata is required for the PostGIS table. This includes information such as the table row count, geometry type and spatial extents of the data in the geometry column. If the table contains a large number of rows determining this metadata is time consuming.&lt;/p&gt;
 &lt;p&gt;By activating this option the following fast table metadata operations are done:&lt;/p&gt;
-&lt;p&gt;1) Row count is determined from table statistics obtained from running the PostgreSQL table analyze function.&lt;/p&gt;
+&lt;p&gt;1) Row count is determined from results of running the PostgreSQL Analyze function on the table.&lt;/p&gt;
```

Comment:
This is not quite the same meaning - I believe the important point the original string is trying to convey that the row count is taken from the statistics table, not calculated exactly.

Reply:
Ok i think i'll revert the changes on this though i might confess i'm not comfortable with the end of the sentence. But i'm not a native speaker and might be translating. 
@sveinki can you please elaborate on what made you report an issue on this sentence in https://github.com/qgis/qgis3_UIX_discussion/issues/19#issuecomment-266994315? Thanks

Reply:
I guess it was the "analyse" vs "analyze" inconsistncy ? Seems right to me now.